pkg/proxy/config/api_test.go

```diff
@@ -84,11 +84,11 @@ func TestServicesFromZero(t *testing.T) {
 		},
 	}
 	services := make(chan ServiceUpdate)
-	source := SourceAPI{servicesWatcher: fakeClient.Services(api.NamespaceAll), endpointsWatcher: fakeClient.Endpoints(api.NamespaceAll), services: services}
+	source := SourceAPI{s: servicesReflector{watcher: fakeClient.Services(api.NamespaceAll), services: services}, e: endpointsReflector{watcher: fakeClient.Endpoints(api.NamespaceAll)}}
```

Comment:
optional nit: make this initialization multi-line, it's hard to read. Multiple occurrences in this file